src/components/Network.vue

```diff
@@ -117,11 +117,11 @@ export default {
         once(event, callback) {
             this.network.moveTo(event, callback);
         },
-        canvasToDom(x, y) {
-            return this.network.canvasToDom(x, y);
+        canvasToDom(p) {
+            return this.network.canvasToDOM(p);
         },
-        DomToCanvas(x, y) {
-            return this.network.DomToCanvas(x, y);
+        DomToCanvas(p) {
```

Comment:
Shouldn't it be either lowerCamelCase `domToCanvas ` either consistent with visjs API `DOMToCanvas` ?

Reply:
Yep, you are right. Changed it to `domToCanvas`.